3_ProtectionProfile/PP_Config-OS.adoc

```diff
@@ -0,0 +1,333 @@
+= PP-Configuration for Protection Profile for General Purpose Operating Systems and collaborative PP-Module for Biometric enrolment and verification - for unlocking the device - [CFG-OS-BIO]
+:showtitle:
+:toc:
+:table-caption: Table
+:revnumber: 0.5
+:revdate: March 20, 2020
+
+== Acknowledgements
+
+This PP-Configuration was developed by the Biometrics Security international Technical Community (BIO-iTC) with representatives from industry, Government agencies, Common Criteria Test Laboratories, and members of academia.
+
+:sectnums:
+:sectnumlevels: 5
+
+== Introduction
+
+The purpose of a PP-Configuration is to define a Target of Evaluation (TOE) that combines Protection Profiles (PPs) and PP-Modules for various technology types into a single configuration that can be evaluated as a whole. The scope includes the definition of the configuration of a computer (a computer in the terms of the PP-Module) that has biometric enrolment and verification capability. The TOE will be defined by a combination of the components described in <<PP-Configuration Components Statement>>.
+
+The computer for the purposes of this PP-Configuration is a system with a General Purpose Operating System plus a Biometric System. The Biometric System may be directly integrated into the physical computer or an external component that has been linked/paired with the physical computer and for which biometric credentials are linked specifically to the paired computer (i.e. templates registered on computer 1 cannot be used on computer 2).
```

Comment:
"... may be directly integrated into the physical computer or an external component that has been linked/paired with the physical computer..." seems to go against what the GPOS PP defines as the TOE. The GPOS TOE includes physical and virtual hardware, so I guess the config should try and allow this too.  Here I Mae a suggestion, but it may be less than perfect...

Suggest "The Biometric System may be directly integrated into the TOE platform, which may include an external component that has been incorporated as part of the TOE platform and for which biometric credentials are linked specifically to the TOE platform (i.e. templates registered on TOE platform 1 cannot be used on TOE platform 2).